data-plane/dispatcher/src/main/java/dev/knative/eventing/kafka/broker/dispatcher/http/HttpConsumerVerticleFactory.java

```diff
@@ -107,54 +110,52 @@ public HttpConsumerVerticleFactory(
     final CircuitBreakerOptions circuitBreakerOptions = createCircuitBreakerOptions(resource);
 
     final var egressDestinationSender = createSender(
-      egress.destination(),
+      egress.getDestination(),
       circuitBreakerOptions,
-      resource.egressConfig()
+      resource.getEgressConfig()
     );
 
-    final var egressConfig = resource.egressConfig();
+    final var egressConfig = resource.getEgressConfig();
```

Comment:
Declare this one before creating `egressDestinationSender`, and pass it to the constructor.